titan/params/classes.yml

```diff
@@ -185,3 +185,25 @@ classes:
     default:
       replace:
         enter_type: replace
+  age_bins:
+    type: definition
+    description: bin of ages, mainly used for assortative mixing
```

Comment:
I'd beef up this description a bit.  Also, this is currently _only_ used for assortative mixing, so I'd harden that language.